src/agent/coverage/src/block.rs

```diff
@@ -78,6 +105,36 @@ impl ModuleCov {
         Self { blocks }
     }
 
+    /// Total count of blocks that have been reached (have a nonzero count).
+    pub fn covered(&self) -> u64 {
+        self.blocks
+            .values()
+            .map(|b| u64::min(1, b.count as u64))
+            .sum()
+    }
+
+    /// Total count of known blocks.
+    pub fn features(&self) -> u64 {
+        self.blocks.len() as u64
+    }
+
+    /// Total count of blocks covered by `self` but not `other`.
```

Comment:
This is implied by your comment but maybe worth noting anyway - a difference of `0` does not imply identical coverage.